hazelcast/include/hazelcast/client/proxy/SerializingProxy.h

```diff
@@ -215,6 +202,25 @@ class HAZELCAST_API SerializingProxy
           });
     }
 
+    template<typename K, typename V>
+    inline boost::future<boost::optional<entry_view<K, V>>>
+    to_object_entry_view(
+      boost::future<boost::optional<map::data_entry_view>> data_future,
+      const K& key)
```

Comment:
```suggestion
      K key)
```
Eliminate copy and use move at caller since most probably that key will not be used after that point on the caller, right?